tmt/steps/provision/mrack.py

```diff
@@ -1039,6 +1077,130 @@ class BeakerGuestData(tmt.steps.provision.GuestSshData):
              """,
     )
 
+    bootc_user: Optional[str] = field(
+        default=None,
+        option=('--bootc-user'),
+        metavar='BOOTC_USER',
+        help="""
+             Specify bootc user.
+             """,
+    )
+
+    bootc_password: Optional[str] = field(
+        default=None,
+        option=('--bootc-password'),
+        metavar='BOOTC_PASSWORD',
+        help="""
+             Specify bootc password.
+             """,
+    )
+
+    base_image_url: str = field(
+        default="quay.io/fedora/fedora-bootc:latest",
+        option=('--base-image-url'),
+        metavar='BASE_IMAGE_URL',
+        help="""
+             Select container image to be used to build the derived bootc image.
+             Cannot be used with container-file.
+             """,
+    )
+
+    test_image_name: str = field(
```

Comment:
`bootc` prefix, `NAME` is enough for the `metavar`, more info in the help text, please.